css/css-display/accessibility/display-contents-role-and-label.html

```diff
@@ -0,0 +1,290 @@
+
+  <!-- Landmarks and regions -->
+  <header style="display: contents;" data-expectedrole="banner" data-testname="header with display: contents has banner role" class="ex-role">x</header>
+  <nav style="display: contents;" data-expectedrole="navigation" data-testname="nav with display: contents has navigation role" class="ex-role">x</nav>
+  <aside style="display: contents;" data-expectedrole="complementary" data-testname="aside with display: contents has complementary role" class="ex-role">x</aside>
+  <main style="display: contents;" data-expectedrole="main" data-testname="main with display: contents has main role" class="ex-role">x</main>
+  <footer style="display: contents;" data-expectedrole="contentinfo" data-testname="footer with display: contents has contentinfo role" class="ex-role">x</footer>
+  <form aria-label="label" style="display: contents;" data-expectedrole="form" data-expectedlabel="label" data-testname="form with display: contents has form role" class="ex-role-and-label">x</form>
+  <search style="display: contents;" data-expectedrole="search" data-testname="search with display: contents has search role" class="ex-role">x</search>
+  <section aria-label="label" style="display: contents;" data-expectedrole="region" data-expectedlabel="label" data-testname="section with aria-label and display: contents has region role" class="ex-role-and-label"></section>
+
+  <div role="banner" style="display: contents;" data-expectedrole="banner" data-testname="div with role banner and display: contents has banner role" class="ex-role">x</div>
+  <div role="navigation" style="display: contents;" data-expectedrole="navigation" data-testname="div with role navigation and display: contents has navigation role" class="ex-role">x</div>
+  <div role="complementary" style="display: contents;" data-expectedrole="complementary" data-testname="div with role complementary and display: contents has complementary role" class="ex-role">x</div>
+  <div role="main" style="display: contents;" data-expectedrole="main" data-testname="div with role main and display: contents has main role" class="ex-role">x</div>
+  <div role="contentinfo" style="display: contents;" data-expectedrole="contentinfo" data-testname="div with role contentinfo and display: contents has contentinfo role" class="ex-role">x</div>
+  <div role="region" aria-label="label" style="display: contents;" data-expectedrole="region" data-expectedlabel="label" data-testname="div with role region, aria-label and display: contents has region role" class="ex-role-and-label">x</div>
+  <div role="form" aria-label="label" style="display: contents;" data-expectedrole="form" data-expectedlabel="label" data-testname="div with role form, aria-label and display: contents has form role" class="ex-role-and-label">x</div>
+  <div role="search" style="display: contents;" data-expectedrole="search" data-testname="div with role search and display: contents has search role" class="ex-role">x</div>
```

Comment:
Nit. These two are the reverse order from their respective host language element group above.

Reply:
Fixed: e9a79c5ff0a99b54ccd0011f3f6a8b663908bfc1.